index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portforio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="/styles.css">
+</head>
+<body>
+ <header>
+    <a href="#home"> <img src="images/My-Logo.png"> </a>  
```

Comment:
Good work making the changes required, kindly add this into the `<nav>` tag. its part of the navigation bar. It's important to keep semantic correct to aid screen readers.

The navbar should be styled using flexbox, have the logo at one end and the hamburger icon at the other just as it is in the figma template:
This is your current design:
<img width="699" alt="Screenshot 2022-10-28 at 11 19 35" src="https://user-images.githubusercontent.com/48546992/198540394-a8c36ed5-242b-4ef2-8204-422c218b288c.png">
This is the figma template:
<img width="369" alt="Screenshot 2022-10-28 at 11 23 07" src="https://user-images.githubusercontent.com/48546992/198540774-629104f3-651f-4cf3-8785-b6e6175b92ee.png">